tests/braille-specs/pa.yaml

```diff
@@ -0,0 +1,2638 @@
+﻿# Punjabi Grade 1
+# Copyright (C) 2022 by Compass Braille <http://www.compassbraille.org>
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
```

Comment:
What I'm missing is a description of where the information in this test comes from. You explained it pretty well in issue https://github.com/liblouis/liblouis/issues/1014 and https://github.com/liblouis/liblouis/pull/1211. Maybe just include something along those lines in here (and possibly in the table files as well).